data/com/mojang/blaze3d/platform/NativeImage.mapping

```diff
@@ -51,29 +68,41 @@ CLASS com/mojang/blaze3d/platform/NativeImage
 	METHOD fromBase64 (Ljava/lang/String;)Lcom/mojang/blaze3d/platform/NativeImage;
 		ARG 0 string
 	METHOD getA (I)I
-		ARG 0 col
+		ARG 0 packedARGB
```

Comment:
It is 'packed' in the sense that instead of 4 separates bytes for each channel (red, green, blue, alpha), they are instead packed into a single integer value. Also, instead of `color` I would suggest `abgr` or `abgrColor` (to be consistent with a comment from me above, I will use `abgrColor`).